.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -59,7 +59,7 @@ To find the version numbers for the three systems below use the following comman
 - `npm list @sequelize/core` / `yarn list --pattern @sequelize/core` (v7 alpha)
 - `node -v`
 - `npm list typescript` / `yarn list --pattern typescript`
-- 'Database' is which actual database system you're using. e.g. 'PostgreSQL 14', 'MariaDB 10.11', etc…
+- 'Database' is which actual database system you're using. e.g. 'PostgreSQL 14', 'MariaDb 10.11', etc…
```

Comment:
```suggestion
- 'Database' is which actual database system you're using. e.g. 'PostgreSQL 14', 'MariaDB 10.11', etc…
```

These usages of MariaDB are referring to the project itself

Reply:
Reverted